portal-impl/src/content/Language.properties

```diff
@@ -3880,6 +3880,7 @@ repositories-require-at-least-one-write-role=Repositories require at least one w
 repository=Repository
 repository-fields=Repository Fields
 repository-id=Repository ID
+reposted-from=Reposted from
```

Comment:
Why do we have to include this here?? I've found that the same key exists in microblogs-web content/Language.properties:3883 so if we include it in the portal-impl we should remove it.